src/reference-designs/tap-architecture-planning.md

```diff
@@ -270,3 +272,8 @@ Logging for Tanzu Application Platform is handled by the upstream Kubernetes int
 | Decision ID   | Design Decision   | Justification | Implication
 |---            |---                |---            |---
 |TAP-009  |Use an external logging platform.          |  An external logging platform will keep logs for the duration of their retention window and offer superior searching capabilities.  | None
+
+
+## Deployment Instructions
+For instructions on how to deploy this reference design, see [Deploy multicluster Tanzu Application Platform profiles](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.2/tap/GUID-multicluster-installing-multicluster.html)
```

Comment:
Please add a period at the end of the sentence.

`For instructions on how to deploy this reference design, see [Deploy multicluster Tanzu Application Platform profiles](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.2/tap/GUID-multicluster-installing-multicluster.html).`